windows/devices/uwb/UwbCxAdapter.cxx

```diff
@@ -0,0 +1,239 @@
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::StsConfiguration arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::StsPacketConfiguration arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::UwbMacAddress arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ arg.GetValue(), service_ddi_map.at(ptag) }));
+                           //    if (ptag == ::uwb::protocol::fira::UwbConfiguration::ParameterTag::ControllerMacAddress) {
+                           //        builder.Push(std::move(UwbAppConfigurationParameter{ arg.GetValue(), service_ddi_map.at(ptag) }));
+                           //    } else if(ptag == ::uwb::protocol::fira::UwbConfiguration::ParameterTag::ControleeShortMacAddress){
+                           //        builder.Push(std::move(UwbAppConfigurationParameter{ arg.GetValue(), service_ddi_map.at(ptag) }));
+                           //    }
+                       },
+                       [&builder, ptag](::uwb::UwbMacAddressFcsType arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::UwbMacAddressType arg) {
+                       },
+                       [&builder, ptag](std::unordered_set<::uwb::protocol::fira::ResultReportConfiguration> arg) {
+                           uint8_t result = 0;
+                           for (auto config : arg) {
+                               switch (config) {
+                               case ::uwb::protocol::fira::ResultReportConfiguration::TofReport: {
+                                   result |= 0x1;
+                                   break;
+                               }
+                               case ::uwb::protocol::fira::ResultReportConfiguration::AoAAzimuthReport: {
+                                   result |= 0x2;
+                                   break;
+                               }
+                               case ::uwb::protocol::fira::ResultReportConfiguration::AoAElevationReport: {
+                                   result |= 0x4;
+                                   break;
+                               }
+                               default: {
+                                   result |= 0x8;
+                                   break;
+                               }
+                               }
```

Comment:
Instead of having these magic numbers, let's update `ResultReportConfiguration` and use the underlying value per below:
```
enum class ResultReportConfiguration : uint8_t {
    TofReport = 0x01U,
    AoAAzimuthReport = 0x02U,
    AoAElevationReport  0x04U,
    AoAFoMReport = 0x08U,
};
```

Then used like this:
```suggestion
                               result |= notstd::to_underlying(config);
```